Story/Episode 2/un_025090.csv

```diff
@@ -11,11 +11,11 @@ un_025090_0080#0,"""2. Call me whatever you like."""
 un_025090_0090#0,"""...a classified mission, huh? I get it, you can't say."""
 un_025090_0100#0,"""But I gotta call you something! Mind if I decide on what?"""
 un_025090_0110#0,"""Can't say your name? What, you on a top secret mission? Hmm, so what should I call you?"""
-un_025090_0120#0,"""'One Punch Man! is too long. Teacher? Nah, not quite. Oh, it's on the tip of my tongue..."""
-un_025090_0130#0,"""I got it! Teacher! You're stronger than anyone I know, so teacher it is!"""
-un_025090_0140#0,"""Thanks for saving me, teacher!"""
+un_025090_0120#0,"""\""One Punch Man\"" is too long. Teacher? Nah, not quite. Oh, it's on the tip of my tongue..."""
+un_025090_0130#0,"""I got it! Master! You're stronger than anyone I know, so \""Master\"" it is!"""
```

Comment:
Shishou is closer to Teacher than master. You could use Master, but in this case, Teacher would fit better.